homeassistant/helpers/event.py

```diff
@@ -1295,7 +1295,12 @@ def utc_converter(utc_now: datetime) -> None:
         """Convert passed in UTC now to local now."""
         hass.async_run_hass_job(job, dt_util.as_local(utc_now))
 
-    return async_track_point_in_utc_time(hass, utc_converter, point_in_time)
+    track_job = HassJob(
+        utc_converter,
+        name=f"UTC converter for {job.name}",
```

Comment:
I would put utc converter at the end since when these get logged you get limited amount of text